vnext/Microsoft.ReactNative/Utils/ResourceBrushUtils.cpp

```diff
@@ -44,6 +43,33 @@ void UpdateResourceBrush(
   }
 }
 
+void UpdateToggleResourceBrush(
+    const xaml::FrameworkElement &element,
+    const std::wstring &resourceName,
+    const xaml::Media::Brush brush) {
+  const auto resources = element.Resources();
+
+  // sanity check for null pointer
+  if (resources != nullptr) {
+    if (brush != nullptr) {
+      auto resourceBrush = resources.Lookup(winrt::box_value(resourceName));
```

Comment:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.resourcedictionary.lookup?view=winrt-22000

"The value, if an item with the specified key exists. Use the HasKey method to determine whether the key exists."